authappliance/pi-appliance

```diff
@@ -283,23 +283,41 @@ class Peer(object):
         sftp = SFTPClient.from_transport(transport)
         for file in self.files:
             sftp.put(file, file)
+
+        # set relevant remote privacyIDEA options
+        remote_config = PrivacyIDEAConfig(DEFAULT_CONFIG, opener=sftp.file)
+        local_config = PrivacyIDEAConfig(DEFAULT_CONFIG)
+        remote_config.config['PI_PEPPER'] = local_config.config['PI_PEPPER']
+        remote_config.config['SUPERUSER_REALM'] = local_config.config['SUPERUSER_REALM']
+        remote_config.save()
```

Comment:
... here I just copy ``PI_PEPPER`` (for the admin passwords, I think) and ``SUPERUSER_REALM`` to the remote pi.cfg.
I decided against copying the whole ``pi.cfg`` (as we did before) because this would set a wrong MySQL password.